docker/Dockerfile.xenial_debug

```diff
@@ -0,0 +1,31 @@
+FROM ubuntu:xenial
+# Base image for debug builds.
+# Built manually uploaded as "istionightly/base_debug"
+
+# Do not add more stuff to this list that isn't small or critically useful.
+# If you occasionally need something on the container do
+# sudo apt-get update && apt-get whichever
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y \
+      curl \
+      iptables \
+      iproute2 \
+      iputils-ping \
+      dnsutils \
+      netcat \
+      tcpdump \
+      net-tools \
+      libc6-dbg gdb \
+      elvis-tiny \
+      lsof \
+      busybox \
+      sudo &&  apt-get upgrade -y && \
+    rm -rf /var/lib/apt/lists/*
+
+# Sudoers used to allow tcpdump and other debug utilities.
+RUN mkdir /var/lib/istio && useradd -m --uid 1337 istio-proxy && \
+    echo "istio-proxy ALL=NOPASSWD: ALL" >> /etc/sudoers && \
+    chown -R istio-proxy /var/lib/istio
+
+
+# TODO: add delve
```

Comment:
missing newline

which target builds it ? will it get built by the release process ?